rts/Net/NetCommands.cpp

```diff
@@ -1291,7 +1291,20 @@ void CGame::ClientReadNet()
 						skirmishAIHandler.RemoveSkirmishAI(aiNum);
 
 						CPlayer::UpdateControlledTeams();
-						eventHandler.PlayerChanged(playerNum);
+						playerHandler.Player(playerNum)->NotifyPlayerChanged();
+
+						LOG("Skirmish AI \"%s\" (ID:%i), which controlled team %i is now dead", aiInstanceName.c_str(), aiNum, aiTeamId);
```

Comment:
rts/Net/NetCommands.cpp:1296:79: error: use of undeclared identifier 'aiInstanceName'

  ...AI \"%s\" (ID:%i), which controlled team %i is now dead", aiInstanceName...